html/pfappserver/lib/pfappserver/Form/Config/Network.pm

```diff
@@ -70,7 +70,7 @@ has_field 'ip_assigned' =>
       option_pattern => sub {
           {
               message => "Must be a list of mac list ip address",
-              regex   => "^[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3}(,[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3})*\$",
+              regex   => "/^[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3}(,[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3})*\$/",
```

Comment:
@jrouzierinverse  Regarding your previous note , i've added back the "" but also kept the / / in order to make it a valid regex and fix the -  "Unrecognized escape \d passed throug" ...

Reply:
Now I understand why you wanted to change that good catch.
The string should be 
```perl
"^[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\\d{1,3}(.\\d{1,3}){3}(,[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\\d{1,3}(.\\d{1,3}){3})*\$"
```
The \d was not escaped properly.

Reply:
@jrouzierinverse Done